modules/odc-installing-web-terminal.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// web_console/odc-about-web-terminal.adoc
+
+[id="odc-installing-web-terminal_{context}"]
+= Installing the web terminal
+
+You can install the web terminal using the *Web Terminal* Operator listed in the OpenShift Container Platform OperatorHub. When you install the *Web Terminal*  Operator, the Custom Resources (CRs) required for the command line configuration, such as the `DevWorkspace` resource, are automatically installed along with the Operator.
```

Comment:
Let me know if I am wrong, but I think, user needs to create the DevWorkspace  CR as it won't get created automatically when user installs the operator.

Reply:
When I tested this, I did not need to install the DevWorkspace CR,  it worked as soon as the operator was installed and I refreshed my screen. @sleshchenko I will test this again, but can you please confirm from your side too?

Reply:
No separate creation of the CR is required by the user. It should automatically happen in the background.

Reply:
Technically, it's not true

> such as the `DevWorkspace` resource, are automatically installed along with the Operator.

Along with Operator CRDs are installed, like DevWorkspace CRDs(Custom Resource Definition), but CRs should be created by users(it's done by OpenShift Console when user opens a terminal)

Reply:
Made changes to point out the distinction between the CRDs and CRs.